ONSdigital/response-operations-ui
# Missing icon exception fix

# Motivation and Context
missing icon was causing 500 errors on load
# What has changed
deleted a line of css that was overriding the icon UR with an incorrect link
# How to test?
go to the survey tab and click a survey with live surveys, icon should be next to CSV download link

# Links
https://trello.com/c/7fNvJ3iq/762-exceptions-being-raised-on-missing-icon

# Screenshots (if appropriate):
